clients/drcachesim/drpt2trace/drpt2trace.cpp

```diff
@@ -64,9 +66,9 @@
 /* A collection of options. */
 static droption_t<bool> op_help(DROPTION_SCOPE_FRONTEND, "help", false,
                                 "Print this message", "Prints the usage message.");
-static droption_t<bool> op_print_trace(DROPTION_SCOPE_FRONTEND, "print_trace", false,
-                                       "Print trace",
-                                       "Print the disassemble code of the trace.");
+static droption_t<bool> op_print_ins(DROPTION_SCOPE_FRONTEND, "print_ins", false,
```

Comment:
For uniformity: s/print_ins/print_instrs/

Reply:
Done.